.travis.yml

```diff
@@ -18,6 +18,7 @@ env:
   - KIND_K8S_VERSION="v1.15.3"
   - KIND_K8S_VERSION="v1.14.1"
   - GOPROXY=https://proxy.golang.org/
```

Comment:
I guess this doesn't matter any more. We can probably drop this line.
One benefit of dropping it is that it reduces the number of concurrent jobs for one PR. IIRC we can only run limited number of concurrent jobs for one project.

Reply:
You can either do it in this PR or in another PR. Up to you.

Reply:
@mengqiy Do you mean `GOPROXY=https://proxy.golang.org/`? That would remove every `osx` test, wouldn't it? The `osx` line under `os` and the `exceptions` for `osx` could also be removed.

Reply:
Would be nice add a comment in the file to make clear that the os run the test.sh and the Linux run the e2e in the vars wdyt?

Reply:
> You can either do it in this PR or in another PR. Up to you.

I think we should do it in another PR, discussing it here is ok

Reply:
@mengqiy Was this what you were thinking about? Each stage is gated behind the previous one, so we are saving resources as those PR that do not pass the linting stage will not take any more resource.